src/utils/build.py

```diff
@@ -0,0 +1,12 @@
+import json
+
+UNKNOWN_BUILD_INFO = {"version": "unknown", "branch": "unknown", "commit": "unknown"}
+
+
+def get_build_info() -> dict:
+    try:
+        with open("../../build-info.json", "r") as f:
```

Comment:
It won't work. Because we are running Oracle from root folder.
I suppose smth like this should work
```suggestion
        with open("./build-info.json", "r") as f:
```

Reply:
fixed